test/service/test_response.py

```diff
@@ -3502,7 +3502,10 @@ def tearDownClass(cls):
 
     @property
     def facets(self) -> Sequence[str]:
-        return self.app_module.app.metadata_plugin.facets
+        plugin = self.app_module.app.metadata_plugin
+        facets = plugin.facets
+        facets[facets.index(plugin.source_id_field)] = 'accessible'
```

Comment:
`.facets` returns an immutable sequence so we should not modify in place.

More importantly, the need for these three lines isn't readily apparent to me, so it may require an explanation in the form of a comment.